app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -741,7 +751,23 @@ public void toggleMoreResultsBox() {
 
     @OnClick({R.id.inputAppraisalExpandBox})
     public void toggleAppraisalBox() {
-        toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        boolean visible = toggleVisibility(inputAppraisalExpandBox, appraisalBox);
+        moveInfoWindow(visible);
+    }
+
+    /**
+     * Move infoLayout window to top/bottom.
+     * @param toTop
+     */
+    private void moveInfoWindow(boolean toTop) {
+        if (toTop) {
+            layoutParams.gravity = Gravity.CENTER | Gravity.TOP;
+            Log.d(TAG, "moveInfoWindow: top");
+        } else {
+            layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
+            Log.d(TAG, "moveInfoWindow: bottom");
+        }
+        windowManager.updateViewLayout(infoLayout, layoutParams);
```

Comment:
I'd expect this doesn't go all the way to the top? That's not enough—on a Nexus 5 one needs to go all the way to the top *and* to hide other widgets for this to work.